checkpoint_ngfw/icon_checkpoint_ngfw/actions/add_access_rule/action.py

```diff
@@ -0,0 +1,52 @@
+import komand
+from .schema import AddAccessRuleInput, AddAccessRuleOutput, Input, Output, Component
+# Custom imports below
+import requests
+
+
+class AddAccessRule(komand.Action):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+                name='add_access_rule',
+                description=Component.DESCRIPTION,
+                input=AddAccessRuleInput(),
+                output=AddAccessRuleOutput())
+
+    def run(self, params={}):
+        url = f"https://{self.connection.server_ip}:{self.connection.server_port}/web_api/add-access-rule"
+
+        # Position will accept an int or string like "top" or "bottom"
+        position_string = params.get(Input.POSITION)
```

Comment:
Because the API expects string or int. 

I believe requests will send this as "top" or 1 (no quotes) when I say json=<object> in requests. 

Leaving this as is.